pom.xml

```diff
@@ -73,7 +73,7 @@
         <version.jjwt>0.11.2</version.jjwt>
         <version.jts>1.19.0</version.jts>
         <version.junit>4.13.2</version.junit>
-        <version.junit.jupiter>5.5.2</version.junit.jupiter>
+        <version.junit.jupiter>5.9.3</version.junit.jupiter>
         <version.junit.vintage>5.5.2</version.junit.vintage>
```

Comment:
Can we keep the junit jupiter and vintage versions in sync, please?

Reply:
Fixed this version. Rebase is making the diff look weird. Checking it out right now.